taxi/views.py

```diff
@@ -28,6 +30,20 @@ def index(request):
     return render(request, "taxi/index.html", context=context)
 
 
+@login_required
+def assign_driver_to_car(request, pk):
+    car = get_object_or_404(Car, pk=pk)
+    car.drivers.add(request.user)
```

Comment:
Ensure that `request.user` is a `Driver` instance before adding it to `car.drivers`. This prevents potential errors if a non-driver user attempts to assign themselves to a car.